modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/security/auth/verifier/NewClass.java

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

        * The class name should end with the interface name
        * The package of the class matches the package of the interface

        

        Refer to documentation: